src/server/controllers/editGameTitle.ts

```diff
@@ -0,0 +1,25 @@
+import RecordedGameModel from "@/db/mongo/model/RecordedGameModel";
+import getMongoClient from "@/db/mongo/mongo-client";
+
+export type editGameTitleParams = {
+  gameTitle: string;
+  gameGuid: string;
+};
+
+export default async function editGameTitle(
+  params: editGameTitleParams
+): Promise<void> {
+  const { gameGuid, gameTitle } = params;
+
+  await getMongoClient();
+  try {
+    const existingGame = await RecordedGameModel.findOne({ gameGuid });
+    if (!existingGame) {
+      throw new Error(`Game with gameGuid ${gameGuid} not found.`);
+    }
+    await existingGame.updateOne({ $set: { gameTitle } });
```

Comment:
This should be rewritten to just do an update by gameGuid. It can throw it it doesn't find a GUID for some reason when updating, but that shouldn't happen. This reduces it to one query instead of two.

Reply:
done